apple-pie/index.html

```diff
@@ -4,36 +4,55 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Apple Pie Recipe</title>
+    <link rel="stylesheet" href="styles/style.css" />
     <!-- Remember to link your styles -->
   </head>
+  <header>
+    <h1>Apple Pie</h1>
+  </header>
   <body>
-    Apple Pie
+    <intro-section class="align-center">
```

Comment:
It would be a better practice to call the section just section and leave the surname for the class:

```html
<section class="intro">
```